frontend/apps/web/components/jobs/SchemaTable/SchemaColumns.tsx

```diff
@@ -291,6 +305,7 @@ export function getSchemaColumns(props: Props): ColumnDef<RowData>[] {
         };
         const isPrimaryKey = constraintHandler.getIsPrimaryKey(key);
         const [isForeignKey, fkCols] = constraintHandler.getIsForeignKey(key);
+        const isUniqueConstraint = constraintHandler.getIsUniqueConstraint(key);
```

Comment:
Make sure to also update the accessorFn to include the `Unique` text if it is unique so that it is filterable/sortable